src/inmanta/protocol/methods.py

```diff
@@ -114,22 +121,22 @@ def create_environment(
     Create a new environment
 
     :param project_id: The id of the project this environment belongs to
-    :param name: The name of the environment
-    :param repository: The url (in git form) of the repository
-    :param branch: The name of the branch in the repository
-    :param environment_id: A unique environment id, if none an id is allocated by the server
+    :param name: The name of the environment.
+    :param repository: Optional. The URL (in git format) of the repository.
```

Comment:
What does `git format` mean?

Reply:
I mean ending in .git. I'm not sure how to express this better

Reply:
The fact that URLs of Git repositories end with `.git` is more a convention than hard requirement. Our API also doesn't enforce it. So I wouldn't mention is in our documentation either.